src/main/scripts/org/reaktivity/specification/nukleus/http_cache/streams/proxy/rfc7234/cache.if.server.returns.503.while.validation/accept/client.rpt

```diff
@@ -0,0 +1,72 @@
+#
+# Copyright 2016-2017 The Reaktivity Project
+#
+# The Reaktivity Project licenses this file to you under the Apache License,
+# version 2.0 (the "License"); you may not use this file except in compliance
+# with the License. You may obtain a copy of the License at:
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+property newProxyAcceptRef ${nukleus:newRouteRef()} # external scope
+property date ${http_cache:date()} # shared property
+property date2 ${http_cache:datePlus(2)} # shared property
+property etag ${http_cache:strongEtag()}
+
+connect await ROUTED_PROXY
+        "nukleus://http-cache/streams/source"
+        option nukleus:route ${newProxyAcceptRef}
+        option nukleus:window 8192
+        option nukleus:transmission "half-duplex"
+
+write nukleus:begin.ext ${http:header(":scheme", "http")}
+                        ${http:header(":method", "GET")}
+                        ${http:header(":path", "/path")}
+                        ${http:header(":authority", "localhost:8080")}
+
+
+connected
+
+write close
+read nukleus:begin.ext ${http:header(":status", "200")}
+                       ${http:header("content-length", "11")}
+                       ${http:header("date", date)}
+                       ${http:header("etag", etag)}
+
+read "hello world"
+
+read closed
+read notify REQUEST_CACHED
+
+# INPUT STREAM 2
+connect await REQUEST_CACHED
+        "nukleus://http-cache/streams/source"
+        option nukleus:route ${newProxyAcceptRef}
+        option nukleus:window 8192
+        option nukleus:transmission "half-duplex"
+
+write nukleus:begin.ext ${http:header(":scheme", "http")}
+                        ${http:header(":method", "GET")}
```

Comment:
I don't see anything in the 2nd request forcing the revalidation, so I would expect this to be served from cache??

Reply:
```
     ${http:header("if-modified-since", date)}
      ${http:header("if-none-match", etag)}
```

Should force it to do validation and archive logs logs shows correct one.

```
[http-cache -> target]	[0x0000000000000001] BEGIN "http-cache" [0x52e3896a9ff9dbbb] [0x8000000000000002] [0x0000000000000000]
:scheme: http
:method: GET
:path: /path
:authority: localhost:8080
[http-cache <- target]	[0x0000000000000001] WINDOW [8192] [0]
[http-cache -> source]	[0x0000000000000002] BEGIN "http-cache" [0x0000000000000000] [0x86ba8873482a1e3b] [0x0000000000000000]
:status: 200
content-length: 11
date: Thu, 30 Nov 2017 11:03:29 GMT
etag: "[B@3efffb91"
[http-cache <- source]	[0x0000000000000002] WINDOW [8192] [0]
[target -> http-cache]	[0x000000004bbd44ff] BEGIN "target" [0x0000000000000000] [0x8000000000000002] [0x0000000000000000]
:status: 200
content-length: 11
date: Thu, 30 Nov 2017 11:03:29 GMT
etag: "[B@3efffb91"
[target <- http-cache]	[0x000000004bbd44ff] WINDOW [8192] [0]
[source -> http-cache]	[0x00000000497c6c7d] BEGIN "source" [0x8000000000000002] [0x86ba8873482a1e3b] [0x0000000000000000]
:scheme: http
:method: GET
:path: /path
:authority: localhost:8080
[http-cache -> target]	[0x0000000000000001] END [0x0000000000000000]
[http-cache <- target]	[0x0000000000000003] WINDOW [8192] [0]
[http-cache -> source]	[0x0000000000000002] DATA [2816] [0x0000000000000000]
[http-cache <- source]	[0x0000000000000002] WINDOW [11] [0]
[target -> http-cache]	[0x000000004bbd44ff] DATA [2816] [0x0000000000000000]
[source -> http-cache]	[0x00000000497c6c7d] END [0x0000000000000000]
[http-cache -> target]	[0x0000000000000003] BEGIN "http-cache" [0x52e3896a9ff9dbbb] [0x8000000000000004] [0x0000000000000000]
:scheme: http
:method: GET
:path: /path
:authority: localhost:8080
if-modified-since: Thu, 30 Nov 2017 11:03:29 GMT
if-none-match: "[B@3efffb91"
[http-cache -> source]	[0x0000000000000002] END [0x0000000000000000]
[http-cache <- source]	[0x0000000000000004] WINDOW [8192] [0]
[target -> http-cache]	[0x000000004bbd44ff] END [0x0000000000000000]
[source -> http-cache]	[0xffffffffdd2bbc73] BEGIN "source" [0x8000000000000002] [0xdd87f4460c9d9f37] [0x0000000000000000]
:scheme: http
:method: GET
:path: /path
:authority: localhost:8080
if-modified-since: Thu, 30 Nov 2017 11:03:29 GMT
if-none-match: "[B@3efffb91"
[http-cache -> target]	[0x0000000000000003] END [0x0000000000000000]
[http-cache -> source]	[0x0000000000000004] BEGIN "http-cache" [0x0000000000000000] [0xdd87f4460c9d9f37] [0x0000000000000000]
:status: 503 (Service Unavailable)
[target -> http-cache]	[0x000000001847f729] BEGIN "target" [0x0000000000000000] [0x8000000000000004] [0x0000000000000000]
:status: 503 (Service Unavailable)
[source -> http-cache]	[0xffffffffdd2bbc73] END [0x0000000000000000]
[http-cache -> source]	[0x0000000000000004] END [0x0000000000000000]
[target -> http-cache]	[0x000000001847f729] END [0x0000000000000000]
```